### PR TITLE
ENH: Python wrapping reads VLV pixel type correctly

### PR DESCRIPTION
**Pull request by dzenanz**
The test file is produced by 3D Slicer, which is a saved
segmentation of mouse tibia (down-sampled for test).

## PR Checklist
- [x] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [x] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)
- [x] Added test (or behavior not changed)
- [ ] Updated API documentation (or API not changed)
- [x] Added [license](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/KWStyle/ITKHeader.h) to new files (if any)
- [x] Added Python wrapping to new files (if any) as described in [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) Section 9.5
- [ ] Added [ITK examples](https://github.com/InsightSoftwareConsortium/ITKSphinxExamples) for all new major features (if any)

### CODE REVIEW

**Comment by dzenanz** [comment]
@thewtex I can't really tell what's wrong:
https://open.cdash.org/test/439396031
https://open.cdash.org/test/439396082
https://open.cdash.org/test/439370061

File: Wrapping/Generators/Python/itk/support/extras.py [hunk 1]
```diff
@@ -976,7 +976,10 @@ def imread(
         # Increase dimension if last dimension is not of size one.
         if increase_dimension and image_IO.GetDimensions(dimension - 1) != 1:
             dimension += 1
-        ImageType = itk.Image[pixel_type, dimension]
+        if itk.template(pixel_type)[0] is itk.VariableLengthVector:
```
Comment (thewtex) [inline Wrapping/Generators/Python/itk/support/extras.py:979]: -> `if isinstance(pixel_type, itk.support.template_class.itkTemplate) and itk.template(pixel_type)[0] is itk.VariableLengthVector`

**Review by thewtex — CHANGES_REQUESTED** [review]
Thanks @dzenanz !

`itk.template` will not work with `itk.CType`. Suggestion inline.

**Review by thewtex — APPROVED** [review]
:snake:

File: Wrapping/Generators/Python/itk/support/extras.py (hunk 1 above)
Comment (dzenanz) [reply Wrapping/Generators/Python/itk/support/extras.py]:
```text
  File "C:\Dev\ITK-py\Wrapping\Generators\Python\itk\support\extras.py", line 983, in imread
    ImageType = itk.Image[pixel_type, dimension]
  File "C:\Dev\ITK-py\Wrapping\Generators\Python\itk\support\template_class.py", line 529, in __getitem__
    raise itk.TemplateTypeError(self, key)
itk.support.extras.TemplateTypeError: itk.Image is not wrapped for input type `itk.VariableLengthVector[itk.UC], int`.
```
Indeed, `itk.Image` is not wrapped for `itk.VariableLengthVector`. Should we add wrapping of VLV for all scalar types? Should we make it configurable via a CMake option?

Comment (thewtex) [reply Wrapping/Generators/Python/itk/support/extras.py]: > ImageType = itk.Image[pixel_type, dimension]

We should never hit this line if `pixel_type` is VLV. VLV corresponds to the pixel type for `itk.VectorImage`. We cannot / should not wrap `itk.Image` for it.

**Review by thewtex — APPROVED** [review]

**Comment by thewtex** [comment]
Added to `release` for 5.2.1

**Pull request merged**